vtex/admin-ui-example
# Update Admin UI

#### What is the purpose of this pull request?

Update Admin UI to the latest version (0.132.7).

#### How to test it?

Navigate to the live demo and make sure it's working as expected: [adminuiexample--teamadmin.myvtex.com/admin/example](https://adminuiexample--teamadmin.myvtex.com/admin/example/)

#### Types of changes
